src/Teams/Repository/TeamMemberRepository.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Teams.Data;
+using Teams.Models;
+
+namespace Teams.Repository
+{
+    public class TeamMemberRepository:IRepository<TeamMember>
+    {
+        private readonly ApplicationDbContext _dbContext;
+
+        public TeamMemberRepository(ApplicationDbContext dbContext)
+        {
+            this._dbContext = dbContext;
+        }
+
+        public async Task<TeamMember> DeleteAsync(TeamMember entity)
```

Comment:
Do we really need to return entity as we already have it as an input parameter? I would suggest to use boolean as a return type just to be insure that the delete operation is successful or not. This will be enough for all cases